src/components/Link.react.js

```diff
@@ -27,6 +27,9 @@ function CustomEvent(event, params) {
 }
 CustomEvent.prototype = window.Event.prototype;
 
+/**
+ * Link allows you to create a clickable link within a multi-page app
+ */
```

Comment:
Do we want to say "for links outside this app, use `html.A` instead"? Or is `Link` good in both cases?